jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -2820,7 +2820,99 @@ public static MethodHandle filterArguments(MethodHandle handle, int startPositio
 		MethodHandle result = FilterArgumentsHandle.get(handle, startPosition, filters, newType);
 		return result;
 	}
+
+/*[IF Java12]*/
+	/**
+	 * Produce a MethodHandle that preprocesses some of the arguments by calling the preprocessor handle.
```

Comment:
Can this be more descriptive? An example would help a lot. I have to read the implementation to guess what the MethodHandle returned by this API is supposed to do. The MethodHandle seems to replace only the argument of `handle` at `foldPosition` with the result of the combiner handle. The combiner handle takes arguments at positions specified by `argumentIndices`.